tools/azure-sdk-tools/devtools_testutils/azure_unit_test.py

```diff
@@ -0,0 +1,176 @@
+from dotenv import load_dotenv, find_dotenv
+import logging
+import os
+import unittest
+
+from azure_devtools.scenario_tests import GeneralNameReplacer, AzureTestError
+# from azure_devtools.scenario_tests.config import TestConfig
+
+from . import mgmt_settings_fake as fake_settings
+from .azure_testcase import get_resource_name, get_qualified_method_name
+from .config import TEST_SETTING_FILENAME
+
+
+class AzureUnitTest(unittest.TestCase):
+    def __init__(self, method_name, config_file=None):
+        super(AzureUnitTest, self).__init__(method_name)
```

Comment:
You don't need this method override if you do nothing else than a super call. Also it seems you're not using config_file?